Revit_oM/Enums/BoundCondition.cs

```diff
@@ -30,14 +30,15 @@ namespace BH.oM.Adapters.Revit.Enums
         "Below is the meaning of each condition for rooms. The same applies to spaces and areas." +
         "- Unplaced: https://help.autodesk.com/view/RVT/2022/ENU/?guid=GUID-BC1DC181-B6D0-4479-8385-363A9EE5E75E" +
         "- Not enclosed: https://help.autodesk.com/view/RVT/2022/ENU/?guid=GUID-1AEDF540-7CB3-4CAB-885A-ACDF70154312" +
-        "- Redundant: https://help.autodesk.com/view/RVT/2022/ENU/?guid=GUID-0DF409DD-3BBD-4488-B544-D075D1807747" +
+        "- Redundant(Overlapping another space): https://help.autodesk.com/view/RVT/2022/ENU/?guid=GUID-0DF409DD-3BBD-4488-B544-D075D1807747" +
```

Comment:
Shouldn't this line start from 'Overlapping:' to follow the convention? Also missing description for no geometry